memcached/memcached.go

```diff
@@ -9,6 +13,42 @@ type CacheClient struct {
 	client *memcache.Client
 }
 
+// GetBytes converts interface{} to a byte array
+func GetBytes(key interface{}) ([]byte, error) {
+	var buf bytes.Buffer
+	enc := gob.NewEncoder(&buf)
```

Comment:
check if this can be singleton

Reply:
This can't be singleton since a byte buffer is passed to it when creating a new one. The data is written to this buffer post encoding